agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/LazyConfigurationAccessor.java

```diff
@@ -148,11 +146,4 @@ static boolean shouldSetConnectionString(boolean lazySetOptIn, String enableAgen
     }
     return false;
   }
-
-  static void setInstrumentationLoggingLevel(String loggingLevel) {
-    if (loggingLevel != null && !loggingLevel.isEmpty()) {
-      Config.get()
-          .updateProperty("otel.experimental.log.capture.threshold", loggingLevel.toUpperCase());
```

Comment:
this doesn't seem to break it.  LogExporter is calling this at runtime  
`String severity = Config.get().getString("otel.experimental.log.capture.threshold");`. I don't think we need to call LogExporter here. I will revert this change.

Reply:
Oh, good point. But you won't be able to revert, because `Config.get().updateProperty()` was a hack in our otel-fork that we needed when `otel.experimental.log.capture.threshold` was read by the instrumentation. I will send a PR to fix properly.